codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -499,6 +500,13 @@ private void notifyHalfClosed(Http2Stream stream) {
             }
         }
 
+        private void initChildrenIfEmpty() {
+            IntObjectMap<DefaultStream> empty = PrimitiveCollections.emptyIntObjectMap();
+            if (children == empty) {
```

Comment:
inline the empty collection for the win?

`PrimitiveCollections.<DefaultStream>emptyIntObjectMap()`